Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -341,19 +341,19 @@ public final class ManifestLoader: ManifestLoaderProtocol {
 
     /// Validate the provided manifest.
     private func validate(_ manifest: Manifest, toolsVersion: ToolsVersion, diagnostics: DiagnosticsEngine?) throws {
-        try validateTargets(manifest, diagnostics: diagnostics)
-        try validateProducts(manifest, diagnostics: diagnostics)
-        try validateDependencies(manifest, toolsVersion: toolsVersion, diagnostics: diagnostics)
+        try self.validateTargets(manifest, diagnostics: diagnostics)
+        try self.validateProducts(manifest, diagnostics: diagnostics)
+        try self.validateDependencies(manifest, toolsVersion: toolsVersion, diagnostics: diagnostics)
```

Comment:
I see there are many lines where an explicit self reference is added. Are they for purely stylistic reasons, or is there a functional aspect to them as well?

Reply:
These were automatically added by [SwiftFormat](https://github.com/nicklockwood/SwiftFormat) according to the rules added by #3012. See also https://github.com/apple/swift-package-manager/pull/3012#discussion_r518025194